app/styles/_project-bar.less

```diff
@@ -19,48 +19,57 @@
   .add-to-project {
     align-items: center;
     color: @navbar-pf-vertical-color;
-    display: flex;
-    flex: 0 0 35px;
     font-weight: 300;
     justify-content: center;
-    @media (min-width: @screen-xs-max) {
-      flex: 1 0 auto;
-      margin-right: 15px;
-    }
-    &:before {
-      background:  @navbar-os-project-menu-border-inset;
-      content: '';
-      display: block;
-      height: 100%;
-      left: 0px;
-      position: absolute;
-      top: 0;
-      width: 1px;
-      @media (min-width: @screen-xs-max) {
-        display: none;
-      }
+    margin: 0 5px 0 0;
+    @media (min-width: @screen-sm-min) {
+      margin: 5px 15px 0 5px;
     }
     > a {
       align-items: center;
       color: @navbar-os-project-menu-color;
       display: flex;
-      padding: 5px 0;
       flex: 1 1 auto;
       justify-content: center;
+      line-height: 1.9em;
```

Comment:
To ensure cross-browser consistency, it might be more reliable to set this to 1 and adjust padding?